python/BioSimSpace/Sandpit/Exscientia/_SireWrappers/_molecule.py

```diff
@@ -518,6 +518,22 @@ def isDecoupled(self):
         else:
             return False
 
+    def isLipid(self):
+        """
+        Whether this molecule is decoupled, i.e. it can be used in a
+        free-energy decoupling simulation.
+
+        Returns
+        -------
+
+        is_decoupled : bool
+            Whether the molecule is decoupled.
+        """
+        if self._sire_object.hasProperty("lipid"):
```

Comment:
Oh, that's already there?

Reply:
Or do we have to change our prep as well?

Reply:
No I added it in bss-wrappers